sources/packages/web/src/components/common/students/StudentFileUploads.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header
+          title="File Uploads"
+          :recordsCount="studentFileUploads?.length"
+        >
+          <template #actions>
+            <slot name="uploadBtn"></slot>
+          </template>
+        </body-header>
+      </template>
+      <content-group>
+        <toggle-content :toggled="!studentFileUploads.length">
+          <DataTable
+            :value="studentFileUploads"
+            :paginator="true"
+            :totalRecords="studentFileUploads?.length"
+            :rows="DEFAULT_PAGE_LIMIT"
+            :rowsPerPageOptions="PAGINATION_LIST"
+          >
+            <Column
+              field="groupName"
+              header="Document Purpose"
+              :sortable="true"
+            ></Column>
+            <Column field="metadata" header="Application #">
+              <template #body="slotProps">{{
+                emptyStringFiller(slotProps.data.metadata?.applicationNumber)
+              }}</template></Column
+            >
+            <Column field="updatedAt" header="Date Submitted"
+              ><template #body="slotProps">{{
+                dateOnlyLongString(slotProps.data.updatedAt)
+              }}</template></Column
+            >
+            <Column field="updatedAt" header="File">
+              <template #body="slotProps">
+                <div
+                  v-if="canDownloadFiles"
+                  class="file-label"
+                  @click="fileUtils.downloadStudentDocument(slotProps.data)"
+                >
+                  <span class="mr-4">
+                    <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
+                  ></span>
+                  <span>{{ slotProps.data.fileName }}</span>
+                </div>
+                <div v-else>
+                  <span class="mr-4">
+                    <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
+                  ></span>
+                  <span>{{ slotProps.data.fileName }}</span>
+                </div>
+              </template></Column
+            >
+          </DataTable>
+        </toggle-content>
+      </content-group>
+      <slot></slot>
```

Comment:
What is the purpose of this default slot? I guess it may be carried over from the old file.

Reply:
Looking further I got this part, 
![image](https://user-images.githubusercontent.com/54600590/233687704-2bf18659-f364-43b5-9aea-14e2a8651dca.png)


But a question here. When a file upload happens, how will the list of files refresh?

Reply:
I am sorry, I didn't get why we need this default slot, is it for the modal? if it's, do we really need it??

Reply:
but I agree with @dheepak-aot, after file upload, the component needs to be refreshed

Reply:
![image](https://user-images.githubusercontent.com/7859295/233711501-e09b0dcb-2d10-4bb1-9701-e7d5a968ef3b.png)

Reply:
I had added the `slot` you are referring to to render any additional AEST or Institution template specific content.